apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -498,45 +536,39 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
 
   // Used in levels with pre-defined "Blockly Variables"
-  blocklyWrapper.WorkspaceSvg.prototype.registerGlobalVariables = function (
-    variableList
-  ) {
+  extendedWorkspaceSvg.registerGlobalVariables = function (variableList) {
     this.globalVariables = variableList;
     this.getVariableMap().addVariables(variableList);
   };
 
-  blocklyWrapper.WorkspaceSvg.prototype.getContainer = function () {
+  extendedWorkspaceSvg.getContainer = function () {
     return this.svgGroup_.parentNode;
   };
 
-  const googleBlocklyBlocklyResize =
-    blocklyWrapper.WorkspaceSvg.prototype.resize;
-  blocklyWrapper.WorkspaceSvg.prototype.resize = function () {
-    googleBlocklyBlocklyResize.call(this);
-    if (cdoUtils.getToolboxType() === ToolboxType.UNCATEGORIZED) {
-      this.flyout_?.resize();
-    }
-  };
-
-  blocklyWrapper.WorkspaceSvg.prototype.events = {
+  extendedWorkspaceSvg.events = {
     dispatchEvent: () => {}, // TODO
   };
 
   // TODO - called by StudioApp, not sure whether they're still needed.
-  blocklyWrapper.WorkspaceSvg.prototype.setEnableToolbox = function (
-    enabled
-  ) {};
-  blocklyWrapper.WorkspaceSvg.prototype.traceOn = function (armed) {};
+  extendedWorkspaceSvg.setEnableToolbox = function () {};
+  extendedWorkspaceSvg.traceOn = function () {};
+
+  const extendedVariableMap = blocklyWrapper.VariableMap
+    .prototype as ExtendedVariableMap;
 
-  blocklyWrapper.VariableMap.prototype.addVariables = function (variableList) {
+  extendedVariableMap.addVariables = function (variableList) {
     variableList.forEach(varName => this.createVariable(varName));
   };
 
-  // TODO - used for spritelab behavior blocks
-  blocklyWrapper.Block.createProcedureDefinitionBlock = function (config) {};
+  // Used for spritelab behavior blocks.
+  // We can remove this once we are ready to no longer support sprite lab on CDO Blockly.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (blocklyWrapper.Block as any).createProcedureDefinitionBlock = function () {};
 
-  // TODO - used to add "create a behavior" button to the toolbox
-  blocklyWrapper.Flyout.configure = function (type, config) {};
+  // In cdo this is used to add "create a behavior" button to the toolbox
+  // Once we have fully moved to Google Blockly we can remove this.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (blocklyWrapper.Flyout as any).configure = function () {};
```

Comment:
Similar to above comment. There's also a reference to this in a dance file that I bet we don't need.